ruby-monday-blog/spec/models/post_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
You can actually shorten this to just `describe Post do` which is cleaner and easier to read.